src/pages/home/report/ReportActionCompose.js

```diff
@@ -591,6 +592,10 @@ class ReportActionCompose extends React.Component {
         const commentAfterColonWithEmojiNameRemoved = this.state.value.slice(this.state.selection.end).replace(CONST.REGEX.EMOJI_REPLACER, CONST.SPACE);
 
         this.updateComment(`${commentBeforeColon}${emojiCode} ${commentAfterColonWithEmojiNameRemoved}`, true);
+        // Clear the contents of a TextInput in some android keyboards
```

Comment:
Clear comments will be helpful for others to understand the issue.

```suggestion

        // In some Android phones keyboard, the text to search for the emoji is not cleared
        // will be added after the user starts typing again on the keyboard. This package is 
        // a workaround to reset the keyboard natively.
```